mock/docs/mock.1

```diff
@@ -104,8 +104,13 @@ the specified destination path.
 Copies the source paths (files or directory trees) from the chroot to
 the specified destination path.
 .TP
-\fB\-\-chroot\fP \fICOMMAND [\fIARGS...\fR]
-Run the specified command non\-interactively within the chroot (no 'clean' is performed). Command output will be sent to the log files. Note that COMMAND is shell expanded only if there are no ARGS. Otherwise, there is no shell expansion.
+\fB\-\-chroot\fP [\fI\-\-\fR] \fICOMMAND [\fIARGS...\fR]
+Run the specified command non\-interactively within the chroot (no
+\fB\-\-clean\fR is performed).
+
+This mode is similar to \fishell\fR mode, except that the output \fBis logged\fR
+and the \fICOMMAND\fR and \fIARGS\fR are not shell expanded in chroot.  See also
```

Comment:
This is the case I spoke about. Some people still expects that `--chroot 'ls *'` will be shell expanded in chroot.

Reply:
And it stays expanded...  but the documentation fix is wrong.  Right?

Reply:
I updated it to:
```
       --chroot [--] COMMAND [ARGS...]
              Run the specified command non-interactively  within  the  chroot
              (no --clean is performed).

              This  mode  is  similar to shell mode, except that the output is
              logged and the COMMAND and ARGS are not shell expanded in chroot
              when the variant with ARGS is used, see the difference:
                  $ mock --quiet --chroot -- echo '*'
                  *
                  $ mock --quiet --chroot -- 'echo *'
                  bin  boot  builddir dev etc home lib lib64 media mnt opt ...
              See also --shell.

```